Add docu for editing chat requests (#787)
eclipse-theia/theia-website: src/docs/user_ai.md

**Hint:** The context file support in Theia IDE shown above is built on the generic context variable capabilities of the underlying Theia AI framework. It therefore can be customized and extended with tool-specific context variable types. See the [Theia AI documentation](/docs/theia_ai) for more details.

### Editing Chat Requests

> Users can modify previously sent messages in the chat UI through the "Edit" option. This functionality is especially helpful for refining prompts, fixing typographical errors, or experimenting with various phrasing without losing context or initiating a new conversation from the beginning.

Reviewer: ```suggestion
Users can modify previously sent messages in the chat UI through the "Edit" option. This functionality is especially helpful for refining prompts if you are unhappy with the LLMs response, or experimenting with various phrasing without losing context or initiating a new conversation from the beginning.
```
